package/pom.xml

```diff
@@ -55,7 +55,7 @@
         <plugins>
             <plugin>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <version>2.5.2</version>
+                <version>3.0.0</version>
```

Comment:
this should probably be using the `maven-assembly.version` property

Reply:
It seemed like properties were used a bit inconsistently. Seems like you are suggesting that we should always use properties? If so, is the motivation to make them easier to update?

Reply:
yeah, they probably are used inconsistently. there are a couple of reasons. first, we ended up in a few places where versions were getting inconsistent. obviously some are not repeated, but others are and it makes updating easier. the other reason is that properties get inherited from parent poms, so defining them is important, but then sticking to only using properties ensures we're consistent everywhere.